
builtin_interfaces/Time stamp

uint8 flight_mode

uint8 INIT = 0
uint8 PRE_ARM = 1
uint8 ARM = 2
uint8 MISSION_START = 3
uint8 MISSION_END = 4
uint8 ABORT = 255

# TOPICS offboard/flight_mode/get offboard/flight_mode/set
